src/sessions/softphone-session-handler.ts

```diff
@@ -101,14 +101,18 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
   }
 
   hasActiveSession (): boolean {
-    return this.activeSession?.state === 'active';
+    return this.activeSession?.peerConnection.connectionState === 'connected';
```

Comment:
This is the main culprit here. For some reason the stanza session still claims it's `active` even though it's not. The `peerConnection` says it's `closed` after waking up from sleep, but the session state says `active`